msteams-platform/graph-permissions/resource-specific-consent.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Resource specific consent in Teams
+description: Describes resource specific consent in Teams and how to make advantage of it.
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: Overview
+keywords: teams authorization OAuth SSO AAD rsc Graph
+---
+# Resource specific consent (RSC) — Microsoft Teams Graph API
+
+Microsoft Teams and Graph API integration enables you to use API endpoints to manage team life cycles within their organizations. When originally introduced, the permissions model was non-granular and required the approval of a global admin. The resource specific consent (RSC) permissions model enables *team owners* to grant consent for an application to access and/or modify a team's data. The granular, Teams-specific, RSC permissions define what an application can do within a specific team:
```

Comment:
I'm not crazy about the opening two sentences.

Reply:
Normally reference pages don't talk about the history of the product, just the present. The other thing that gives me pause is the team lifecycle analogy -- RSC doesn't help you create the team, so it's a bit of a stretch to sell it as a team lifecycle solution. -- and "automated channel lifecycle" undersells was going to be the most widely used RSC feature -- reading channel messages. That said, I do like that picture :) . Here's a PowerPoint version, it's very easy to edit. https://microsoft.sharepoint.com/:p:/t/skypespacesteamnew/EZRSQv3xORtLgAfkBW8IB1sBS7ycZGgz9HKhmNkUaat0dg?e=q6EZWF&nav=eyJzSWQiOjI4NCwiY0lkIjoyMjc5OTcxOTE3fQ